### ISSUE TITLE
Issues with the Sentinel slide

### ISSUE THREAD

**Issue by joeworkman** [malsup/cycle2#182]
The sentinel slide was a very clever idea. However, it stops to work properly when the slide copied has styles associated with the ID for that slide. I am not sure what a solution could be though since you obviously cannot have 2 elements in the DOM with the same ID.

Am I making sense? Let me know and I can provide more details.

**Comment by malsup** [comment]
Right, that's a limitation of the default behavior.  The id attr is removed from the sentinel.  But you can choose any of the slides to be the sentinel, or choose other auto-sizing behavior.  Or roll your own auto-sizing logic.

http://jquery.malsup.com/cycle2/api/#auto-height
